src/plugins/grpc/client.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const FORMAT_HTTP_HEADERS = require('opentracing').FORMAT_HTTP_HEADERS
+const Tags = require('opentracing').Tags
+
+function getMethodType (definition) {
+  if (definition.requestStream) {
+    if (definition.responseStream) {
+      return 'bidi'
+    }
+
+    return 'client_stream'
+  }
+
+  if (definition.responseStream) {
+    return 'server_stream'
+  }
+
+  return 'unary'
+}
+
+function patch (grpc, tracer, config) {
+  if (config.client === false) return
+
+  config = config.client || config
+
+  function datadogInterceptor (options, nextCall) {
+    const path = options.method_definition.path
+    const methodType = getMethodType(options.method_definition)
+    const scope = tracer.scopeManager().active()
+    const span = tracer.startSpan('grpc.client', {
+      childOf: scope && scope.span(),
+      tags: {
+        [Tags.SPAN_KIND]: Tags.SPAN_KIND_RPC_CLIENT,
+        'grpc.method.name': path,
+        'grpc.method.type': methodType,
+        'resource.name': path.split('/').slice(-1),
+        'service.name': config.service || `${tracer._service}-grpc-client`
+      }
+    })
+
+    return new grpc.InterceptingCall(nextCall(options), {
+      sendMessage: (message, next) => {
+        if (config.message_fields) {
```

Comment:
Our standard for keys is camelCase. However, I think just `fields` would be enough since fields only exist on messages.